src/Umbraco.Web.UI.NetCore/Program.cs

```diff
@@ -7,18 +8,20 @@ namespace Umbraco.Cms.Web.UI.NetCore
     public class Program
     {
         public static void Main(string[] args)
-        {
-            CreateHostBuilder(args)
+            => CreateHostBuilder(args)
                 .Build()
                 .Run();
-        }
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
-                .ConfigureLogging(x =>
-                {
-                    x.ClearProviders();
-                })
-                .ConfigureWebHostDefaults(webBuilder => { webBuilder.UseStartup<Startup>(); });
+#if DEBUG
+                .ConfigureAppConfiguration(config
+                    => config.AddJsonFile(
+                            "appsettings.Local.json",
```

Comment:
@Shazwazza Any reason to use a capital L?  I think it would only matter for Linux developers, but they are also the ones who are notoriously lazy about capitalization.

Reply:
IMO capital L is correct. It follows the convention from: `appsettings.Production.json` and `appsettings.Development.json`.

Reply:
Yep but i guess the .gitignore doesn't have the correct case, i'll fix